lms/djangoapps/certificates/tests/test_signals.py

```diff
@@ -440,78 +433,3 @@ def test_verified_to_audit(self):
         ) as mock_allowlist_task:
             self.verified_enrollment.change_mode('audit')
             mock_allowlist_task.assert_not_called()
-
-
-class CertificateEventBusTests(ModuleStoreTestCase):
```

Comment:
[non-blocking question] I can't think of anything, but I'm wondering how we might ensure that people would know that a signal is in use by the event bus, and that it can't just be deleted or refactored? Maybe I should create a separate issue for this?